app/assets/javascripts/govuk-component/govspeak-enhance-youtube-video-links.js

```diff
@@ -40,6 +40,7 @@
     }
 
     function enhanceYoutubeVideoLinks($el) {
+      if ($($el).hasClass('disable-youtube')) return ;
       $el.find("a[href*='youtube.com'], a[href*='youtu.be']").each(function (i){
```

Comment:
I think the function expects to receive a Jquery selection rather than a selector so maybe no need to convert it again with `$($el)`?